### DISCUSSION TITLE
Identify books that Audible has removed?

### DISCUSSION THREAD

**Discussion by jeffarndt** [openaudible/openaudible#541] [Ideas]
It would be nice if I could use OpenAudible to identify books that are no longer available to download from Audible's library, books that they no longer sell.

Occasionally I come across one of these books, and contact Audible support to get my credit back. These are hard to identify and I find them mostly by accident. Since I have been using Open Audible for a while now, it would be great if there were some type of diff that could be done and flag the old books in a library.

**Comment by oboe45** [comment]
I find, like you, books that seem to be "discontinued" by Audible.  Most times, the actual book is still available to download, but there is usually a better version - better reader or audio, new cover etc.  

I find that when I run Open Audible, most of these books download, but the descriptive text or the cover image is missing.  These are easy to pick out from the generated web age from OA.  Depending upon the issue, I sometimes ask Audible to provide a credit - which they never refuse, or, I update the generated M4a, M4b or mp3 with a new cover page and/or descriptive text and delete the old book from the OA directory and regenerate the web page.  

My Audible membership stretches back probably 15 years and includes time with the US and Canada organizations.  Surprisingly I have had few issues except for the poor quality of some of the audio.  This OA program is complex - especially when importing non-audible books for fixing problems like missing book covers, but it works remarkably well for such a cheap product.

**Comment by openaudible** [comment]
We don't have plans to try to figure out what books have been discontinued. That isn't anything we have much control over. Sorry for the late reply.
